packages/web-components/src/components/va-table/va-table-inner/test/va-table-inner.e2e.ts

```diff
@@ -6,8 +6,11 @@ import { dateSort } from '../../sort/date';
 import { _getCompareFunc } from '../../sort/utils';
 
 describe('va-table', () => {
-  function makeTable() {
-    return `<va-table table-title="this is a caption">
+  function makeTable(props = {}) {
+    const defaultProps = {...props, 'table-title': 'this is a caption'};
+    return `<va-table ${Object.entries(defaultProps)
+      .map(([key, value]) => `${key}="${value}"`)
+      .join(' ')}>
```

Comment:
I saw an opportunity to make it easier to test props without having to copy/paste table markup. We should be able to use this `makeTable` function now with customizable properties.

Reply:
This looks great!